### PR TITLE
chore: bump sdk version

### PR DESCRIPTION
**Pull request by kgpayne**
Related to https://github.com/meltano/squared/issues/432

### CODE REVIEW

**Comment by tayloramurphy** [comment]
@kgpayne @pnadolny13 is this safe to merge?

File: tap_slack/client.py [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+from typing import Any, Dict, List, Optional, Text
 
-from singer_sdk.streams import RESTStream
+from requests import Response
 from singer_sdk.authenticators import BearerTokenAuthenticator
+from singer_sdk.pagination import JSONPathPaginator
+from singer_sdk.streams import RESTStream
+
+
+class ThrottledJSONPathPaginator(JSONPathPaginator):
```
Comment (pnadolny13) [inline tap_slack/client.py:14]: @kgpayne I'm assuming this is to keep status quo based on new pagination logic in the SDK, is that correct?

**Comment by pnadolny13** [comment]
> @kgpayne @pnadolny13 is this safe to merge?

I think so, just clarifying what the throttle paginator changes are doing but if its not changing functionality then it looks good to me!

File: tap_slack/client.py (hunk 1 above)
Comment (kgpayne) [reply tap_slack/client.py:14]: @pnadolny13 correct 🙂 IIRC I had an issue with the existing paginator that was resolved by reimplementing the throttling with a new-style paginator. However the logic didn’t change 👍

**Review by pnadolny13 — APPROVED** [review]

File: tap_slack/client.py (hunk 1 above)
Comment (pnadolny13) [reply tap_slack/client.py:14]: @kgpayne awesome, thanks for clarifying! I also tested in Squared and it works as expected.

**Pull request merged**